### ISSUE TITLE
Batch upscale -> image scale is not working as expected

### ISSUE THREAD

**Issue by Deepa-Aru** [upscayl/upscayl#681]
### Checklist

- [X] I have **tried ALL** of the steps mentioned in the [troubleshooting guide](https://github.com/upscayl/upscayl/wiki/Troubleshooting).
- [X] I have **searched** for my issue in the [issues tab](https://github.com/upscayl/upscayl/issues?q=).
- [ ] My error involves vkEnumeratePhysicalDevices or vkCreateInstance, which means I should ignore the other issues and file this one despite the last checkbox.

### Describe the Bug

Single image upscale is working as expected. When I tried with batch upscale, The image scale is not working as expected.

### Scenario 1 : NOT Working
A folder contains 5 images of dimension 1024x1024 jpeg images. I have selected batch upscale with below settings.
Save Image as -> PNG
Image Scale -> 2x
Upon upscale, the processed output images are 4x scaled even though the output folder named as `upscayl_realesrgan-x4plus_x2`. Each image dimension is 4096x4096.

### Scenario 2 : Working
Tried the same with 1 or 2 images in the folder, its working as expected. 

I am using this application from 2.9.5. And I am able to reproduce the issue on both 2.9.5 and 2.9.8.

### Steps
- Have a folder contains 5 images of dimension 1024x1024 jpeg images.
- Verify the settings as 
  - Save Image as -> PNG
  - Image Scale -> 2x
- Select batch upscale
- Upscale the images. 
- Expected output should be 2048x2048 but actual image sizes are 4096x4096.

### Upscayl Version (or commit hash)

2.9.8

### Platform

Windows

### OS Version

Windows 11

### GPU Name

RTX 3050

### Logs

```sh
⚙️ Getting saveImageAs from localStorage:  png
⚙️ Getting model from localStorage:  {"label":"General Photo (Real-ESRGAN)","value":"realesrgan-x4plus"}
⚙️ Getting gpuId from localStorage:  1
🔀 Setting model to 
🔀 Setting model to realesrgan-x4plus
🐞 BACKEND REPORTED:  💾 Updating Save Output Folder:  false
🔄 Resetting image paths
🐞 BACKEND REPORTED:  📁 Updating Folder Path:  D:\Assets\images\inin
🖼 Selected Folder Path:  D:\Assets\images\inin
🐞 BACKEND REPORTED:  📁 Selected Folder Path:  D:\Assets\images\inin
🐞 BACKEND REPORTED:  📁 Updating Folder Path:  D:\Assets\images\inin
🗂 Setting Output Path:  D:\Assets\images\inin
🧠 Remembering Output Folder:  D:\Assets\images\inin
🐞 BACKEND REPORTED:  📁 Selected Folder Path:  D:\Assets\images\inin
🔄 Resetting Upscaled Image Path
🏁 FOLDER_UPSCAYL
🐞 BACKEND REPORTED:  🖼️ Updating No Image Processing:  false
🐞 BACKEND REPORTED:  📐 Updating Compression:  0
🐞 BACKEND REPORTED:  📢 Upscayl Command:  -i,D:\Assets\images\inin,-o,D:\Assets\images\inin\upscayl_realesrgan-x4plus_x2,-s,4,-m,C:\Program Files\Upscayl\resources\models,-n,realesrgan-x4plus,-g,1,-f,png
🐞 BACKEND REPORTED:  🛑 Updating Stopped:  false
🚧 FOLDER_UPSCAYL_PROGRESS:  [0 Intel(R) RaptorLake-S Mobile Graphics Controller]  queueC=0[1]  queueG=0[1]  queueT=0[1]
[0 Intel(R) RaptorLake-S Mobile Graphics Controller]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[0 Intel(R) RaptorLake-S Mobile Graphics Controller]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[0 Intel(R) RaptorLake-S Mobile Graphics Controller]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1

🚧 FOLDER_UPSCAYL_PROGRESS:  [1 NVIDIA GeForce RTX 3050 6GB Laptop GPU]  queueC=2[8]  queueG=0[16]  queueT=1[2]
[1 NVIDIA GeForce RTX 3050 6GB Laptop GPU]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[1 NVIDIA GeForce RTX 3050 6GB Laptop GPU]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[1 NVIDIA GeForce RTX 3050 6GB Laptop GPU]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1

🚧 FOLDER_UPSCAYL_PROGRESS:  0.00%

🚧 FOLDER_UPSCAYL_PROGRESS:  0.00%

🚧 FOLDER_UPSCAYL_PROGRESS:  2.78%

🚧 FOLDER_UPSCAYL_PROGRESS:  2.78%

🚧 FOLDER_UPSCAYL_PROGRESS:  5.56%

🚧 FOLDER_UPSCAYL_PROGRESS:  5.56%

🚧 FOLDER_UPSCAYL_PROGRESS:  8.33%

🚧 FOLDER_UPSCAYL_PROGRESS:  8.33%

🚧 FOLDER_UPSCAYL_PROGRESS:  11.11%

🚧 FOLDER_UPSCAYL_PROGRESS:  11.11%

🚧 FOLDER_UPSCAYL_PROGRESS:  13.89%

🚧 FOLDER_UPSCAYL_PROGRESS:  13.89%

🚧 FOLDER_UPSCAYL_PROGRESS:  16.67%

🚧 FOLDER_UPSCAYL_PROGRESS:  16.67%

🚧 FOLDER_UPSCAYL_PROGRESS:  19.44%

🚧 FOLDER_UPSCAYL_PROGRESS:  19.44%

🚧 FOLDER_UPSCAYL_PROGRESS:  22.22%

🚧 FOLDER_UPSCAYL_PROGRESS:  22.22%

🚧 FOLDER_UPSCAYL_PROGRESS:  25.00%

🚧 FOLDER_UPSCAYL_PROGRESS:  25.00%

🚧 FOLDER_UPSCAYL_PROGRESS:  27.78%

🚧 FOLDER_UPSCAYL_PROGRESS:  27.78%

🚧 FOLDER_UPSCAYL_PROGRESS:  30.56%

🚧 FOLDER_UPSCAYL_PROGRESS:  30.56%

🚧 FOLDER_UPSCAYL_PROGRESS:  33.33%

🚧 FOLDER_UPSCAYL_PROGRESS:  33.33%

🚧 FOLDER_UPSCAYL_PROGRESS:  36.11%

🚧 FOLDER_UPSCAYL_PROGRESS:  36.11%

🚧 FOLDER_UPSCAYL_PROGRESS:  38.89%

🚧 FOLDER_UPSCAYL_PROGRESS:  38.89%

🚧 FOLDER_UPSCAYL_PROGRESS:  41.67%

🚧 FOLDER_UPSCAYL_PROGRESS:  41.67%

🚧 FOLDER_UPSCAYL_PROGRESS:  44.44%

🚧 FOLDER_UPSCAYL_PROGRESS:  44.44%

🚧 FOLDER_UPSCAYL_PROGRESS:  47.22%

🚧 FOLDER_UPSCAYL_PROGRESS:  47.22%

🚧 FOLDER_UPSCAYL_PROGRESS:  50.00%

🚧 FOLDER_UPSCAYL_PROGRESS:  50.00%

🚧 FOLDER_UPSCAYL_PROGRESS:  52.78%

🚧 FOLDER_UPSCAYL_PROGRESS:  52.78%

🚧 FOLDER_UPSCAYL_PROGRESS:  55.56%

🚧 FOLDER_UPSCAYL_PROGRESS:  55.56%

🚧 FOLDER_UPSCAYL_PROGRESS:  58.33%

🚧 FOLDER_UPSCAYL_PROGRESS:  58.33%

🚧 FOLDER_UPSCAYL_PROGRESS:  61.11%

🚧 FOLDER_UPSCAYL_PROGRESS:  61.11%

🚧 FOLDER_UPSCAYL_PROGRESS:  63.89%

🚧 FOLDER_UPSCAYL_PROGRESS:  63.89%

🚧 FOLDER_UPSCAYL_PROGRESS:  66.67%

🚧 FOLDER_UPSCAYL_PROGRESS:  66.67%

🚧 FOLDER_UPSCAYL_PROGRESS:  69.44%

🚧 FOLDER_UPSCAYL_PROGRESS:  69.44%

🚧 FOLDER_UPSCAYL_PROGRESS:  72.22%

🚧 FOLDER_UPSCAYL_PROGRESS:  72.22%

🚧 FOLDER_UPSCAYL_PROGRESS:  75.00%

🚧 FOLDER_UPSCAYL_PROGRESS:  75.00%

🚧 FOLDER_UPSCAYL_PROGRESS:  77.78%

🚧 FOLDER_UPSCAYL_PROGRESS:  77.78%

🚧 FOLDER_UPSCAYL_PROGRESS:  80.56%

🚧 FOLDER_UPSCAYL_PROGRESS:  80.56%

🚧 FOLDER_UPSCAYL_PROGRESS:  83.33%

🚧 FOLDER_UPSCAYL_PROGRESS:  83.33%

🚧 FOLDER_UPSCAYL_PROGRESS:  86.11%

🚧 FOLDER_UPSCAYL_PROGRESS:  86.11%

🚧 FOLDER_UPSCAYL_PROGRESS:  88.89%

🚧 FOLDER_UPSCAYL_PROGRESS:  88.89%

🚧 FOLDER_UPSCAYL_PROGRESS:  91.67%

🚧 FOLDER_UPSCAYL_PROGRESS:  91.67%

🚧 FOLDER_UPSCAYL_PROGRESS:  94.44%

🚧 FOLDER_UPSCAYL_PROGRESS:  94.44%

🚧 FOLDER_UPSCAYL_PROGRESS:  97.22%

🚧 FOLDER_UPSCAYL_PROGRESS:  97.22%

🚧 FOLDER_UPSCAYL_PROGRESS:  0.00%

🚧 FOLDER_UPSCAYL_PROGRESS:  Upscayl Successful

🚧 FOLDER_UPSCAYL_PROGRESS:  Upscayl Successful

🚧 FOLDER_UPSCAYL_PROGRESS:  0.00%

🚧 FOLDER_UPSCAYL_PROGRESS:  2.78%

🚧 FOLDER_UPSCAYL_PROGRESS:  2.78%

🚧 FOLDER_UPSCAYL_PROGRESS:  5.56%

🚧 FOLDER_UPSCAYL_PROGRESS:  5.56%

🚧 FOLDER_UPSCAYL_PROGRESS:  8.33%

🚧 FOLDER_UPSCAYL_PROGRESS:  8.33%

🚧 FOLDER_UPSCAYL_PROGRESS:  11.11%

🚧 FOLDER_UPSCAYL_PROGRESS:  11.11%

🚧 FOLDER_UPSCAYL_PROGRESS:  13.89%

🚧 FOLDER_UPSCAYL_PROGRESS:  13.89%

🚧 FOLDER_UPSCAYL_PROGRESS:  16.67%

🚧 FOLDER_UPSCAYL_PROGRESS:  16.67%

🚧 FOLDER_UPSCAYL_PROGRESS:  19.44%

🚧 FOLDER_UPSCAYL_PROGRESS:  19.44%

🚧 FOLDER_UPSCAYL_PROGRESS:  22.22%

🚧 FOLDER_UPSCAYL_PROGRESS:  22.22%

🚧 FOLDER_UPSCAYL_PROGRESS:  25.00%

🚧 FOLDER_UPSCAYL_PROGRESS:  25.00%

🚧 FOLDER_UPSCAYL_PROGRESS:  27.78%

🚧 FOLDER_UPSCAYL_PROGRESS:  27.78%

🚧 FOLDER_UPSCAYL_PROGRESS:  30.56%

🚧 FOLDER_UPSCAYL_PROGRESS:  30.56%

🚧 FOLDER_UPSCAYL_PROGRESS:  33.33%

🚧 FOLDER_UPSCAYL_PROGRESS:  33.33%

🚧 FOLDER_UPSCAYL_PROGRESS:  36.11%

🚧 FOLDER_UPSCAYL_PROGRESS:  36.11%

🚧 FOLDER_UPSCAYL_PROGRESS:  38.89%

🚧 FOLDER_UPSCAYL_PROGRESS:  38.89%

🚧 FOLDER_UPSCAYL_PROGRESS:  41.67%

🚧 FOLDER_UPSCAYL_PROGRESS:  41.67%

🚧 FOLDER_UPSCAYL_PROGRESS:  44.44%

🚧 FOLDER_UPSCAYL_PROGRESS:  44.44%

🚧 FOLDER_UPSCAYL_PROGRESS:  47.22%

🚧 FOLDER_UPSCAYL_PROGRESS:  47.22%

🚧 FOLDER_UPSCAYL_PROGRESS:  50.00%

🚧 FOLDER_UPSCAYL_PROGRESS:  50.00%

🚧 FOLDER_UPSCAYL_PROGRESS:  52.78%

🚧 FOLDER_UPSCAYL_PROGRESS:  52.78%

🚧 FOLDER_UPSCAYL_PROGRESS:  55.56%

🚧 FOLDER_UPSCAYL_PROGRESS:  55.56%

🚧 FOLDER_UPSCAYL_PROGRESS:  58.33%

🚧 FOLDER_UPSCAYL_PROGRESS:  58.33%

🚧 FOLDER_UPSCAYL_PROGRESS:  61.11%

🚧 FOLDER_UPSCAYL_PROGRESS:  61.11%

🚧 FOLDER_UPSCAYL_PROGRESS:  63.89%

🚧 FOLDER_UPSCAYL_PROGRESS:  63.89%

🚧 FOLDER_UPSCAYL_PROGRESS:  66.67%

🚧 FOLDER_UPSCAYL_PROGRESS:  66.67%

🚧 FOLDER_UPSCAYL_PROGRESS:  69.44%

🚧 FOLDER_UPSCAYL_PROGRESS:  69.44%

🚧 FOLDER_UPSCAYL_PROGRESS:  72.22%

🚧 FOLDER_UPSCAYL_PROGRESS:  72.22%

🚧 FOLDER_UPSCAYL_PROGRESS:  75.00%

🚧 FOLDER_UPSCAYL_PROGRESS:  75.00%

🚧 FOLDER_UPSCAYL_PROGRESS:  77.78%

🚧 FOLDER_UPSCAYL_PROGRESS:  77.78%

🚧 FOLDER_UPSCAYL_PROGRESS:  80.56%

🚧 FOLDER_UPSCAYL_PROGRESS:  80.56%

🚧 FOLDER_UPSCAYL_PROGRESS:  83.33%
83.33%

🚧 FOLDER_UPSCAYL_PROGRESS:  86.11%

🚧 FOLDER_UPSCAYL_PROGRESS:  86.11%

🚧 FOLDER_UPSCAYL_PROGRESS:  88.89%
88.89%

🚧 FOLDER_UPSCAYL_PROGRESS:  91.67%
91.67%

🚧 FOLDER_UPSCAYL_PROGRESS:  94.44%

🚧 FOLDER_UPSCAYL_PROGRESS:  94.44%

🚧 FOLDER_UPSCAYL_PROGRESS:  97.22%

🚧 FOLDER_UPSCAYL_PROGRESS:  97.22%

🚧 FOLDER_UPSCAYL_PROGRESS:  0.00%

🚧 FOLDER_UPSCAYL_PROGRESS:  Upscayl Successful

🚧 FOLDER_UPSCAYL_PROGRESS:  Upscayl Successful

🚧 FOLDER_UPSCAYL_PROGRESS:  2.78%

🚧 FOLDER_UPSCAYL_PROGRESS:  5.56%

🚧 FOLDER_UPSCAYL_PROGRESS:  8.33%

🚧 FOLDER_UPSCAYL_PROGRESS:  11.11%

🚧 FOLDER_UPSCAYL_PROGRESS:  13.89%

⚙️ Getting saveImageAs from localStorage:  png
⚙️ Getting model from localStorage:  {"label":"General Photo (Real-ESRGAN)","value":"realesrgan-x4plus"}
⚙️ Getting gpuId from localStorage:  1
⚙️ Getting rememberOutputFolder from localStorage:  false
🚧 FOLDER_UPSCAYL_PROGRESS:  16.67%

🚧 FOLDER_UPSCAYL_PROGRESS:  19.44%

🚧 FOLDER_UPSCAYL_PROGRESS:  22.22%

🚧 FOLDER_UPSCAYL_PROGRESS:  25.00%

🚧 FOLDER_UPSCAYL_PROGRESS:  27.78%

🚧 FOLDER_UPSCAYL_PROGRESS:  30.56%

🚧 FOLDER_UPSCAYL_PROGRESS:  33.33%

🚧 FOLDER_UPSCAYL_PROGRESS:  36.11%

🚧 FOLDER_UPSCAYL_PROGRESS:  38.89%

🚧 FOLDER_UPSCAYL_PROGRESS:  41.67%

🚧 FOLDER_UPSCAYL_PROGRESS:  44.44%

🚧 FOLDER_UPSCAYL_PROGRESS:  47.22%

🚧 FOLDER_UPSCAYL_PROGRESS:  50.00%

🚧 FOLDER_UPSCAYL_PROGRESS:  52.78%

🚧 FOLDER_UPSCAYL_PROGRESS:  55.56%

🚧 FOLDER_UPSCAYL_PROGRESS:  58.33%

🚧 FOLDER_UPSCAYL_PROGRESS:  61.11%

🚧 FOLDER_UPSCAYL_PROGRESS:  63.89%

🚧 FOLDER_UPSCAYL_PROGRESS:  66.67%

🚧 FOLDER_UPSCAYL_PROGRESS:  69.44%

🚧 FOLDER_UPSCAYL_PROGRESS:  72.22%

🛑 Stopping Upscayl
🔄 Resetting image paths
🐞 BACKEND REPORTED:  🛑 Updating Stopped:  true
🐞 BACKEND REPORTED:  🛑 Stopping Upscaling Process 4796
⚙️ Getting saveImageAs from localStorage:  png
⚙️ Getting model from localStorage:  {"label":"General Photo (Real-ESRGAN)","value":"realesrgan-x4plus"}
⚙️ Getting gpuId from localStorage:  1
🔀 Setting model to 
🔀 Setting model to realesrgan-x4plus
⚙️ Getting saveImageAs from localStorage:  png
⚙️ Getting model from localStorage:  {"label":"General Photo (Real-ESRGAN)","value":"realesrgan-x4plus"}
⚙️ Getting gpuId from localStorage:  1
⚙️ Getting rememberOutputFolder from localStorage:  false
⚙️ Getting saveImageAs from localStorage:  png
⚙️ Getting model from localStorage:  {"label":"General Photo (Real-ESRGAN)","value":"realesrgan-x4plus"}
⚙️ Getting gpuId from localStorage:  1
🔀 Setting model to 
🔀 Setting model to realesrgan-x4plus
⚙️ Getting saveImageAs from localStorage:  png
⚙️ Getting model from localStorage:  {"label":"General Photo (Real-ESRGAN)","value":"realesrgan-x4plus"}
⚙️ Getting gpuId from localStorage:  1
⚙️ Getting rememberOutputFolder from localStorage:  false
```

**Comment by NayamAmarshe** [comment]
Q1. Why are you stopping the Upscayl process midway?

Q2. Did you read the info text that appears under the batch upscayl progress indicator?

**Comment by Joackk** [comment]
hi, have you solved your problem? I have the same problem

**Comment by aaronliu0130** [comment]
@Joackk Did you wait for the thing to finish?

**Comment by asik** [comment]
Seeing this on 2.10.0.

**Comment by asik** [comment]
If I wait for it to finish, it resizes everything back to its original size, even though I don't have that option selected and I specified a 2X zoom.

**Comment by aaronliu0130** [comment]
You mean the dimensions are equivalent? Are you sure you're looking at the output folder?

**Comment by asik** [comment]
Yup, it's like it just copy pasted the original files into the output folder. If it's relevant, I set the output folder path as a subfolder of the original folder.

**Comment by aaronliu0130** [comment]
Could you take a screenshot of the zoom factor you set

**Comment by asik** [comment]
Settings (all default except Image scale set to 2X)
![image](https://github.com/upscayl/upscayl/assets/1631157/4a4ccbb3-ff5a-4d3b-9886-8cd5e31ddc48)

Hmm I'm having a hard time finding what's the minimal repro from default settings. I tried with a folder with 5 small images and it  worked as expected. But I saw it do this on a folder with more and larger images. Unfortunately this is super time consuming (my laptop is a potato) so it might be a while before I figure out a pattern here.

**Comment by asik** [comment]
Oh wait I gave you some wrong information. Actually when that happened, they were resized using a custom width that I had previously used with the Custom Output Width option. It just happened to be close to the original resolution of these assets so I had assumed they were resized back to the original resolution.

So it's like it stuck with the custom width I had set before even though I deactivated that option, restarted and used the slider instead.
